packages/request-network-sdk/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "@requestnetwork/request-network-sdk",
+  "version": "1.0.0",
+  "publishConfig": {
+    "access": "public"
+  },
+  "description": "Requet Network SDK Library",
+  "keywords": [
+    "requestnetwork",
+    "request-network-sdk"
+  ],
+  "engines": {
+    "node": ">=16.0.0"
+  },
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "directories": {
+    "lib": "src"
+  },
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "build": "tsc -b tsconfig.build.json"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/RequestNetwork/requestNetwork.git"
+  },
+  "homepage": "https://github.com/RequestNetwork/requestNetwork/tree/master/packages/request-network-sdk.js#readme",
+  "bugs": {
+    "url": "https://github.com/RequestNetwork/requestNetwork/issues"
+  },
+  "license": "MIT",
+  "dependencies": {
```

Comment:
the references in here should match the version in the `package.json` of these packages. Right now, you are resolving the version on NPM, not the monorepo's version

Reply:
Hi @benjlevesque thanks for this info, it really helped. Turned out it was because I used the @next modifier (as this has been promoted since I started).

Changed to the correct version(s) and all it well.

The latest commit also includes tsconfig changes to produce es6 output.